src/config/config.ts

```diff
@@ -19,14 +19,17 @@ import client from "../shared/api/cbioportalClientInstance";
 import internalClient from "../shared/api/cbioportalInternalClientInstance";
 import $ from "jquery";
 import {AppStore} from "../AppStore";
-import {proxyAllPostMethodsOnClient} from "../shared/lib/proxyPost";
+import {proxyAllPostMethodsOnClient} from "../public-lib/lib/proxyPost";
 import CBioPortalAPI from "../shared/api/generated/CBioPortalAPI";
 import CBioPortalAPIInternal from "../shared/api/generated/CBioPortalAPIInternal";
 import CivicAPI from "../shared/api/CivicAPI";
 import Genome2StructureAPI from "../public-lib/api/generated/Genome2StructureAPI";
 import GenomeNexusAPI from "../public-lib/api/generated/GenomeNexusAPI";
 import GenomeNexusAPIInternal from "../public-lib/api/generated/GenomeNexusAPIInternal";
 import OncoKbAPI from "../public-lib/api/generated/OncoKbAPI";
+import AppConfig from "appConfig";
+import {sendSentryMessage} from "../shared/lib/tracking";
+import {log} from "../shared/lib/consoleLog";
```

Comment:
These dependencies are removed from the file `proxyPost.ts`, so we now need to include them here in the `config.ts`